app/main.py

```diff
@@ -4,4 +1,34 @@
+import os
+import requests
+
+BASE_URL = "http://api.weatherapi.com/v1/current.json"
+DEFAULT_CITY = "Paris"
+
+
+def get_weather(city: str) -> None:
+    api_key = os.getenv("API_KEY")
+    if not api_key:
+        raise ValueError(
+            "API_KEY is not set. Please provide it as an environment variable"
+        )
+
+    url = f"{BASE_URL}?key={api_key}&q={DEFAULT_CITY}"
+
+    response = requests.get(url)
+    if response.status_code == 200:
+        data = response.json()
+        print(f"Perform request to Weather API for city {DEFAULT_CITY}...")
+
+        city = data["location"]["name"]
+        country = data["location"]["country"]
+        datatime = data["location"]["localtime"]
```

Comment:
There is a typo in the variable name `datatime`. It should be `datetime` to correctly represent the data being stored.